src/components/header/Header.tsx

```diff
@@ -40,27 +62,24 @@ const Header = () => {
       </RouterLink>
 
       <Stack direction='row' spacing={2} marginLeft='auto'>
-        <Link
-          href='https://github.com/msdio/stackticon'
-          sx={{
-            textDecoration: 'none',
-          }}
-        >
+        <GithubLink href='https://github.com/msdio/stackticon'>
```

Comment:
I think there were a lot of parts using the stackticon github URL! (Especially in the saving as img / saving in md format!)
How about using this part as a constant?

Reply:
Good point 🙂